.github/workflows/ci.yaml

```diff
@@ -819,7 +819,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ${{ fromJson(needs.info.outputs.python_versions) }}
-    name: Split tests for full run ${{ matrix.python-version }}
+    name: Split tests for full run Python ${{ matrix.python-version }}
```

Comment:
Small unrelated change missed in #126936. _Not worth the separate PR._

This brings the naming in line with the other pytest job names
```diff
 Run tests Python 3.12 (1)
 Run mariadb:10.3.32 tests Python 3.12
 Run postgres:12.14 tests Python 3.12

-Split tests for full run 3.12
+Split tests for full run Python 3.12
```